projects/showcase/src/app/components/grid/showcase-grid.component.ts

```diff
@@ -1,8 +1,8 @@
 import { Component, OnInit } from '@angular/core';
-import { GridContextMenuOption } from 'systelab-components';
-import { GridContextMenuActionData } from 'systelab-components';
 import { ShowcaseData } from './showcase-grid.model';
 import { ShowcaseGridUtil } from './showcase-grid.util';
+import { GridContextMenuOption } from '../../../../../systelab-components/src/lib/grid/contextmenu/grid-context-menu-option';
```

Comment:
Why not from import { GridContextMenuOption } from 'systelab-components';

Reply:
You are completely right.  

I had some problems at the beginning and the import item has not offered me the option.

Thanks for your comment.